extensions/applicationinsights-react-js/test/ReactAI.test.ts

```diff
@@ -91,7 +91,7 @@ describe("ReactAI", () => {
     };
     core.initialize(config, [reactPlugin, channel]);
     // Mock logger
-    const loggerMock = reactPlugin["_logger"].throwInternal = jest.fn();
+    const loggerMock = core["logger"].throwInternal = jest.fn();
```

Comment:
Can you please explain why logger is update to core["logger"], shouldn't it be reactPlugin["log()"] or something? Thanks

Reply:
the syntax core["xxx"] is an index lookup for the property and could also be writter as core.xxx.
This can't be changed to "log()" as that is not a valid property name (core["log"] would be the correct syntax -- but that would return a reference to the function and not the property).

As internally the log() function uses the "core.logger" property and this unit test is adding a mock implementation of the throwInternal function, this needs to be as defined. So that when the log() function is called it will correctly call the mocked logger implementation.

This change was required as I'm starting to slowly hide the internal properties to provide better minification support.